python/iceberg/parquet/arrow_table_utils.py

```diff
@@ -0,0 +1,203 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Note that most of this here also gets implemented in Arrow at the moment, so you might be able to get rid of this agains in 3-4 months.